### PR TITLE
Add wattpad-scraper

### PR DESCRIPTION
**Pull request by shhossain**
## Wattapad Scraper
If you have any problems let me know.

### CODE REVIEW

File: _data/projects/wattpad-scraper.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Your Project Name Here
```
Comment (ritwik12) [inline _data/projects/wattpad-scraper.yml]: @shhossain This should be `wattpad-scraper`

File: _data/projects/wattpad-scraper.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+site: https://github.com/shhossain/wattpad-scraper
+tags:
+  - python
+  - package
+  - scraper
+  - wattpad
+  - test
+upforgrabs:
+  name: help wanted
+  link: https://github.com/shhossain/wattpad-scraper/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```
Comment (ritwik12) [inline _data/projects/wattpad-scraper.yml]: @shhossain Please change this to https://github.com/shhossain/wattpad-scraper/labels/help%20wanted

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/wattpad-scraper.yml (hunk 1 above)
Comment (shhossain) [reply _data/projects/wattpad-scraper.yml]: name changed

File: _data/projects/wattpad-scraper.yml (hunk 2 above)
Comment (shhossain) [reply _data/projects/wattpad-scraper.yml]: link fixed

**Pull request closed without merging**